Mad_Bash/Assets/Import/FightOrFlight/Scripts/ButtonPressContext.cs

```diff
@@ -51,9 +54,14 @@ private void OnEnable()
         TotalScore = 0;
         Interval.Value = StateTransitionInterval.ToString();
     }
-    
+
     public void UpdateContext()
     {
+        if (stateTransitionInterval == 0)
+            OnTimerEnd.Raise();
+        if (stateTransitionInterval == 1)
```

Comment:
why 1? isn't this what the interval is originally set back to when it starts counting down?

Reply:
@jeremybarzas